cmd/textile/main.go

```diff
@@ -143,8 +147,13 @@ var rootCmd = &cobra.Command{
 			}
 		}
 
+		target := configViper.GetString("api_target")
+		var creds credentials.TransportCredentials
+		if strings.Contains(target, "443") {
```

Comment:
Eh, not the best solution here... I'm sure it will break under some usage. For now, if the target has the normal TLS port, we try to use TLS creds.

Reply:
:shrug: